packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -196,116 +233,60 @@ function ListItem< Item >( {
 				'is-selected': isSelected,
 				'is-hovered': isHovered,
 			} ) }
-			onMouseEnter={ handleMouseEnter }
-			onMouseLeave={ handleMouseLeave }
+			onMouseEnter={ handleHover }
+			onMouseLeave={ handleHover }
 		>
-			<HStack
-				className="dataviews-view-list__item-wrapper"
-				alignment="center"
-				spacing={ 0 }
-			>
+			<HStack className="dataviews-view-list__item-wrapper" spacing={ 0 }>
 				<div role="gridcell">
 					<Composite.Item
-						render={ <div /> }
-						role="button"
```

Comment:
Is there a specific reason for removing the `role` ?

Reply:
Because removing `render={ <div /> }` makes the element a `button`. I debated doing it anyway, because it meant having to override more style rules for the `button` but seems better to not recreate a `button` from a `div` if we don’t need to.

Reply:
Ah got it. I had initially missed the removal of the `render` prop